mod/photos.php

```diff
@@ -175,19 +176,14 @@ function photos_post(App $a)
 		killme();
 	}
 
-	$r = q("SELECT `contact`.*, `user`.`nickname` FROM `contact` LEFT JOIN `user` ON `user`.`uid` = `contact`.`uid`
-		WHERE `user`.`uid` = %d AND `self` = 1 LIMIT 1",
-		intval($page_owner_uid)
-	);
+	$owner_record = User::getOwnerDataById($page_owner_uid);
```

Comment:
Shouldn't this function be renamed `getContactDataById` since it draws most of its data from the contact table?

Reply:
No, "getContactDataById" would be misleading. We are getting the "self" contact plus some parts from the "user" table - which is called "$owner" at most parts of the code. Of course we could name this function "getSelfContactAndRelevantUserDataForGivenUserID" but I think this would be a little bit long :grinning:

Reply:
I don’t find it misleading since it’s in the `Model\User` class but you may be right.